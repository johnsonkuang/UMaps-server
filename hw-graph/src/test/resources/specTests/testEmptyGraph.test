# Test Empty Graph
# Test whether ListNodes work with empty graphs

# Create graph
CreateGraph graph1

# List its nodes
ListNodes graph1