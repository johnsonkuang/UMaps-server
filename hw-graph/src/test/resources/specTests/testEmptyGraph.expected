# Test Empty Graph
# Test whether ListNodes work with empty graphs

# Create graph
created graph graph1

# List its nodes
graph1 contains: